submissions/description/cross_site_scripting_xss/cross_site_request_forgery_csrf/flash_based/high_impact/template.md

```diff
@@ -0,0 +1,48 @@
+# Flash-Based Cross-Site Request Forgery (High Impact)
+
+## Overview of the Vulnerability
+
+Cross-Site Request Forgery (CSRF) occurs when requests to the application are submitted on behalf of an authenticated user without their knowledge via crafted, malicious code which can be in the form of a link the user clicks. The application is unable to distinguish between the malicious request and a legitimate request from the user.
+
+A high impact flash-based CSRF is possible for this application, allowing an attacker to submit requests to the application on behalf of an authenticated user. Additionally, the attacker needs to socially engineer the user to click on a link, or paste the malicious code into the user’s browser. If successful, the code will execute within that user’s browser in the context of this domain.
+
+## Business Impact
+
+CSRF could lead to data theft through the attacker’s ability to manipulate data through their access to the application, and their ability to interact with other users, including performing other malicious attacks which would appear to originate from a legitimate user. These malicious actions could also result in reputational damage for the business through the impact to customers’ trust.
+
+## Steps to Reproduce
+
+1. Enable a HTTP interception proxy, such as Burp Suite or OWASP ZAP
+1. Use a browser to sign into the application at: {{URL}}
+1. Navigate to the following URL and submit the form: {{URL}}
+1. Use the HTTP interception proxy to intercept the request triggered by the form
+1. Observe that a request is sent using `.SWF` file:
+
+{{screenshot}}
+
```

Comment:
This seems redundant for triage, at the end of the day, we only require the SWF file, and sometimes a link, not the creation.